lib/Dialect/AIEVec/Transforms/VectorToAIEVecConversions.cpp

```diff
@@ -2369,10 +2335,9 @@ struct LowerVectorToAIEVec
     auto op = getOperation();
     MLIRContext *context = &getContext();
     RewritePatternSet patterns(context);
-    ConversionTarget target(*context);
     AIEArch aieVersion = AIEArch::AIE;
     if (!aieTarget.empty()) {
-      std::string target = aieTarget;
+      std::string target = aieTarget.getValue();
```

Comment:
There's an implicit downcast but better to be explicit (the actual clang-tidy reco was about 8 bytes discarded or something like that).